ui/src/copainput.tsx

```diff
@@ -87,23 +116,24 @@ export function CopaInput(props: any) {
         renderInput={(params) =>
           <TextField
             {...params}
-            label="Image"
+            label={selectImageLabel}
             error={selectedImageError}
             helperText={selectedImageHelperText}
           />}
       />
       <FormControl fullWidth>
-        <InputLabel id="demo-simple-select-label">Scanner</InputLabel>
+        <InputLabel id="demo-simple-select-label" variant='outlined'>Scanner</InputLabel>
         <Select
           labelId="demo-simple-select-label"
           id="demo-simple-select"
           value={props.selectedScanner}
-          label="Age"
+          label="Scanner"
           onChange={(event: SelectChangeEvent) => {
             props.setSelectedScanner(event.target.value as string);
           }}
         >
           <MenuItem value={"trivy"}>Trivy</MenuItem>
+          <MenuItem value={"no-scanner"}>No Scanner</MenuItem>
         </Select>
```

Comment:
I added a follow up (#8) to create a separate PR to support it, we can remove it from here for now